apps/artist/routes.coffee

```diff
@@ -67,3 +67,13 @@ sd = require('sharify').data
     error: res.backboneError
     success: ->
       res.redirect "/artist/#{req.params.id}"
+
+@ctaPayoff = (req, res) ->
```

Comment:
We'll add a basic metaphysics bootstrap fetch here, since we want to 'seed' the widget not with the homepage followed artists, but with some recommended/related artists to the current one.